docs/articles/documentation/using-testcafe/common-concepts/using-browsers.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: docs
+title: Using Browsers
+permalink: /documentation/using-testcafe/common-concepts/using-browsers.html
+---
+# Using Browsers
+
+TestCafe comes with a [built-in library](#built-in-library) and several [plugins](#browser-provider-plugins) that allow you to easily set up testing in different web browsers.
+
+## Built-in Library
```

Comment:
I have no idea what this section doing here. It should belong to browser provider development section. Also, this lib shouldn't be presented as built-in. Yes, TestCafe requires it, but custom providers should install it as own dependency.